src/main/java/mate/academy/spring/model/dto/request/UserRequestDto.java

```diff
@@ -0,0 +1,40 @@
+package mate.academy.spring.model.dto.request;
+
+public class UserRequestDto {
+    private Long id;
+    private String email;
+    private String password;
+    private byte[] salt;
```

Comment:
```suggestion
public class UserRequestDto {
    private String email;
    private String password;
```
id and salt are generated by our app